io.beekeeperstudio.Studio.yml

```diff
@@ -0,0 +1,72 @@
+app-id: io.beekeeperstudio.Studio
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: beekeeper-studio
+separate-locales: false
+rename-icon: beekeeper-studio
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: beekeeper-studio
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches: [x86_64]
+        url: https://github.com/beekeeper-studio/beekeeper-studio/releases/download/v3.7.10/Beekeeper-Studio-3.7.10.AppImage
+        sha512: 08324cdf9fe124e965a19c448c313af34a5fa148ff82c78c9401e62133e89163aa4733610acfb9c938a853f4d62f1b03756a57cd4beb19e5fd331f6149c142c8
+        dest_filename: Beekeeper-Studio.AppImage
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/beekeeper-studio/beekeeper-studio/releases/latest
+          version-query: .tag_name | sub("v"; "")
+          url-query:
+            .assets[] | select(.name == "Beekeeper-Studio-" + $version + ".AppImage")
+            | .browser_download_url
+      - type: file
+        only-arches: [aarch64]
+
+        url: https://github.com/beekeeper-studio/beekeeper-studio/releases/download/v3.7.10/Beekeeper-Studio-3.7.10-arm64.AppImage
+        sha512: e64c7686313c14894053f7078727cf0cf9ab1edc921e61bb27e6e38c08b0b9a1cec00199a041e04170f289decc6c9ff73861215091feffcb56406c64fa8fd6d2
+        dest_filename: Beekeeper-Studio.AppImage
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/beekeeper-studio/beekeeper-studio/releases/latest
+          version-query: .tag_name | sub("v"; "")
+          url-query:
+            .assets[] | select(.name == "Beekeeper-Studio-" + $version + "-arm64.AppImage")
+            | .browser_download_url
+
+      - type: file
+        path: beekeeper-studio.sh
+
+      - type: file
+        path: io.beekeeperstudio.Studio.metainfo.xml
+
+    build-commands:
```

Comment:
It doesn't really matter, but by convention, `sources` are the last element of a module.